src/main/java/org/lsc/plugins/connectors/james/JamesDao.java

```diff
@@ -145,7 +152,7 @@ public boolean removeAliases(User user, List<Alias> aliasesToRemove) {
 		return aliasesToRemove.stream()
 			.reduce(true,
 				(result, alias) -> result && removeAlias(user, alias),
-				(result1, result2) -> result1 && result2); 
+				(result1, result2) -> result1 && result2);
```

Comment:
what is it?

Reply:
Just Copy and paste reformatting (remove redundant space before). Pls skip this.